src/routes/docs/advanced/security/+page.markdoc

```diff
@@ -29,9 +29,8 @@ Coming soon
 {% /cards_item %}
 
 {% cards_item href="/docs/advanced/security/hippa" title="HIPPA" %}
-Coming soon.
+Appwrite is HIPPA compliant. Learn about how Appwrite's measures to protect PHI.
```

Comment:
What is PHI can we first list the full term?

Reply:
Done, list as full term.

It's also used in the HIPAA docs, but we introduce the term.